### PR TITLE
docs(configure): clarify email report presets

### PR DESCRIPTION
**Pull request by thislg**

### CODE REVIEW

File: docs/configure.md [hunk 1]
```diff
@@ -59,21 +59,21 @@ label: my_import
...
 
-### Options inutilisées
+### Unused options
 
 | Name    | Mandatory (default)   | Type    |     |
 |---------|-----------------------| ------- | --- |
 | `label` | ➖ (`null`)            | ?string |     |
 
-### archive
+### Archive
```
Comment (lucasmirloup) [inline docs/configure.md:68]: I kept the option's name case in the title, that's why it was in lowercase.
Your call :+1:

File: docs/configure.md [hunk 2]
```diff
@@ -59,21 +59,21 @@ label: my_import
...
 
 | Name        | Mandatory (default)    | Type    |                                                                             |
 | ---------- |------------------------| ------- |-----------------------------------------------------------------------------|
 | `dir`      | ➖ (`null`)             | ?string | Path to the folder where import files are stored in the event of quarantine |
 | `enabled`  | ➖ (`true`)             | boolean | Activate/deactivate archiving                                               |
 | `rotation` | ➖ (`30`)               | integer | Number of archiving files to be kept                                        |
 
-### quarantine
+### Quarantine
```
Comment (lucasmirloup) [inline docs/configure.md:76]: Same as above.

File: docs/configure.md [hunk 3]
```diff
@@ -86,21 +86,21 @@ label: my_import
...
-| `recipients`      | ➖ (`[]`)                                | string[] &#124; string | Email(s) of recipient(s)            |
-| `subject_pattern` | ➖ (`[%status%] Import report : %name%`) | string                 | Mail subject                        |
-| `email_template`  | ➖ (`null`)                              | ?string                | Path to a Twig email template file  |
+| Name              | Mandatory (default)                      | Type                   |                                                                                    |
+| ----------------- | ---------------------------------------- | ---------------------- | ---------------------------------------------------------------------------------- |
+| `email_from`      | ✅                                       | ?string                | Sender's email address                                                             |
+| `recipients`      | ➖ (`[]`)                                | string[] &#124; string | Email(s) of recipient(s) or preset name defined in bundle configuration            |
```
Comment (lucasmirloup) [inline docs/configure.md:97]: Suggestion : add a (relative) Markdown link to https://github.com/le-phare/import-bundle/blob/master/docs/configure.md#bundle-configuration

File: docs/configure.md [hunk 4]
```diff
@@ -86,21 +86,21 @@ label: my_import
...
+| `subject_pattern` | ➖ (`[%status%] Import report : %name%`) | string                 | Email subject. Placeholders %name% and %status% are replaced automatically         |
```
Comment (lucasmirloup) [inline docs/configure.md:98]:
```suggestion
| `subject_pattern` | ➖ (`[%status%] Import report : %name%`) | string                 | Email subject. Placeholders `%name%` and `%status%` are replaced automatically         |
```

**Review by lucasmirloup — CHANGES_REQUESTED** [review]

File: docs/configure.md (hunk 1 above)
Comment (thislg) [reply docs/configure.md:68]: Fair enough

**Review by afrancois-lephare — APPROVED** [review]

**Review by lucasmirloup — APPROVED** [review]

**Pull request merged**